raster/src/main/scala/geotrellis/raster/render/ColorRamp.scala

```diff
@@ -125,8 +125,21 @@ object ColorRamp {
     * @param n       Length of list to return
     */
   def spread(colors: Vector[Int], n: Int): Vector[Int] = {
+    /* The requested spread matches the original */
     if (colors.length == n) return colors
 
+    /* 2016 May 20 @ 14:21
```

Comment:
I'd prefer to keep them out. As you say, git should take care of keeping track of changes and times and provides a place to dive through if that info is needed.